src/distributions/cholmvnormal.jl

```diff
@@ -0,0 +1,39 @@
+## Traditional Cholesky representation where Γ is Lower Triangular
+struct CholMvNormal{T, Tμ<:AbstractVector{T}, TΓ<:LowerTriangular{T}} <: AbstractPosteriorMvNormal{T}
+    dim::Int
+    μ::Tμ
+    Γ::TΓ
+    function CholMvNormal(μ::AbstractVector{T}, Γ::LowerTriangular{T}) where {T}
+        length(μ) == size(Γ, 1) || throw(DimensionMismatch("μ and Γ have incompatible sizes")) 
+        new{T,typeof(μ),typeof(Γ)}(length(μ), μ, Γ)
+    end
+    function CholMvNormal(
+        dim::Int,
+        μ::Tμ,
+        Γ::TΓ
+    ) where {
+        T,
+        Tμ<:AbstractVector{T},
+        TΓ<:LowerTriangular{T},
+    }
+        length(μ) == size(Γ, 1) || throw(DimensionMismatch("μ and Γ have incompatible sizes")) 
+        new{T,Tμ,TΓ}(dim, μ, Γ)
+    end
+end
+
+Distributions.cov(d::CholMvNormal) = XXt(d.Γ)
+Distributions.entropy(d::CholMvNormal) = logdet(d.Γ) + 0.5 * length(d) * log(2π) 
+
+
+function reparametrize!(x, q::CholMvNormal, z)
+    x .= q.μ .+ q.Γ * z
+end
+
+
+function to_vec(q::CholMvNormal)
+    vcat(q.μ, vec(q.Γ))
+end
```

Comment:
Can we not make use of something similar to `Flux.params` here since we already depend on `Functors.jl`?

Reply:
Well you need to tell that to ForwardDiff.jl, ReverseDiff.jl and co :D

Reply:
There are no ties between `Flux.params` and the AD-framework used anymore:)
It's using Functors.jl to define what's trainable and what isn't. So `Flux.params` should *just work* for the above distributions. Then you just `vec` and `vcat`.

Reply:
Ah sorry I misunderstood you, I thought you were complaining about the `vcat`.